server_development/topics/providers.adoc

```diff
@@ -181,24 +162,11 @@ For example `HostnameProvider` specifies the hostname to be used by {project_nam
 Hence there can be only single implementation of this provider active for the {project_name} server. If there are multiple provider implementations available to the server runtime,
```

Comment:
We can probably remove all the text related to configuring a default provider because it is documented at https://www.keycloak.org/server/configuration-provider.

Reply:
Focus here is to remove the WildFly documentation, not to enhance the Quarkus documentation. So this should be handled in a follow-up.